deploy/src/main/resources/README.md

```diff
@@ -10,106 +10,85 @@ The Job Service is a RESTful Web Service and provides a simple API.  It tracks t
 
 ## Demonstration
-The Docker Compose file contains the following services:
+The deployment files contain the following services:
 
 ![](images/job-service-deploy.png)
 
 1. Job Service  
     This is the Job Service itself.  As discussed it is a RESTful Web Service and is the primary service being demonstrated here.
 
-    By default port 9411 is used to communicate with the Job Service but if that port is not available then the `JOB_SERVICE_PORT` environment variable can be set to have a different port used.
-
-2. Job Service Database  
-    Internally the Job Service uses a PostgreSQL Database to store the Job Status information.  When the stack is started a `job-service-db` volume will be created to store the database files.
-
-3. RabbitMQ  
+2. RabbitMQ  
     The Worker Framework is a pluggable infrastructure and technically it can use different messaging systems.  However it is most common for RabbitMQ to be used for messaging, and that is what is used here.
 
-4. Job Tracking Worker  
+3. Job Tracking Worker  
     For simplicity the Job Tracking Worker is not shown on the diagram above.  The diagram shows messages passing directly between the workers, but in reality the messages are passed through the Job Tracking Worker, which acts as a proxy for them.  It routes them to their intended destination but it also updates the Job Service Database with the progress.  This means that the Job Service is able to provide accurate progress reports when they are requested.
 
-5. Job Service Scheduled Executor  
+4. Job Service Scheduled Executor  
     This is a polling service that identifies jobs in the system that depend on other jobs which are now complete. It is an ExecutorService which schedules a task to execute repeatedly identifying jobs which are ready to run. For simplicity, this service is not shown in the diagram but for each job identified, a message is then published on RabbitMQ in order to start the job.
 
-6. GlobFilter Worker  
+5. GlobFilter Worker  
     This is a simple worker developed just for this demonstration.  It is a Batch Worker which takes in a glob-pattern as the Batch Definition.  Glob-patterns are generally fairly simple.  For example, `*.txt` means "all text files in the input folder".  Even more complex patterns like `**/t*.txt`, which means "all text files which start with the letter 't' and are in the input folder or in any subfolders of the input folder", are fairly easy to understand.  The worker produces a separate task for each file which matches the glob-pattern.
+    
+    The input folder has been set to `/job-service-test/input-files` for the purposes of this demonstration.  This directory will be created on your Kubernetes Cluster when the Job Service components have been deployed.
```

Comment:
Working through it, I still don't feel we've got this right.  If there is a cluster of machines I don't think we know which one the input files will be read from and which the output files will be written to.

I want to introduce a simple web-based file server into the deployment - this one looks like it might fit the bill:
https://docs.filebrowser.xyz/installation#docker

Then we can update the usage instructions to have the user upload the sample files, and download the result files.

Reply:
FileBrowser added to the deployment along with usage instructions updated.